core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -507,6 +510,113 @@ public LogicalPlan visitDedupe(Dedupe node, AnalysisContext context) {
         consecutive);
   }
 
+  /** Build {@link LogicalLookup}. */
+  @Override
+  public LogicalPlan visitLookup(Lookup node, AnalysisContext queryContext) {
+    LogicalPlan child = node.getChild().get(0).accept(this, queryContext);
+    List<Argument> options = node.getOptions();
+    // Todo, refactor the option.
```

Comment:
Why is this a TODO?

Reply:
This is just copied from https://github.com/opensearch-project/sql/blob/43263963aca2ada949d55ae01cd38788d5d96fbb/core/src/main/java/org/opensearch/sql/analysis/Analyzer.java#L495 and not added by us. Seems there is a geneal need to refactor the options - but thats not specific to this PR